PayloadPkg/PayloadPkg.dec

```diff
@@ -33,10 +33,14 @@
 
 [PcdsFixedAtBuild, PcdsPatchableInModule]
   # These will be patched in the FDF file.
-  gPayloadTokenSpaceGuid.PcdPayloadHobList    | 0x00000000 | UINT32 | 0x10001000
-  gPayloadTokenSpaceGuid.PcdPayloadStackSize  | 0x00000000 | UINT32 | 0x10001002
-  gPayloadTokenSpaceGuid.PcdPayloadHeapSize   | 0x00000000 | UINT32 | 0x10001003
-  gPayloadTokenSpaceGuid.PcdGlobalDataAddress | 0x00000000 | UINT32 | 0x10001004
+  gPayloadTokenSpaceGuid.PcdPayloadHobList    		| 0x00000000 | UINT32 | 0x10001000
+  gPayloadTokenSpaceGuid.PcdPayloadStackSize  		| 0x00000000 | UINT32 | 0x10001002
+  gPayloadTokenSpaceGuid.PcdPayloadHeapSize   		| 0x00000000 | UINT32 | 0x10001003
+  gPayloadTokenSpaceGuid.PcdGlobalDataAddress 	 	| 0x00000000 | UINT32 | 0x10001004
```

Comment:
Not sure why we change these lines, better only have necessary changes.

Reply:
Done

Reply:
Done